sdk/java/pulumi/src/main/java/com/pulumi/Context.java

```diff
@@ -21,6 +21,13 @@ public interface Context extends OutputContext, ConfigContext {
      */
     String stackName();
 
+    /**
+     * @return the name of the current organization
+     */
+    default String organizationName() {
```

Comment:
I decided to provide a default implementation out of an abundance of caution (here and on `Deployment`). I could see an argument for not doing it, though, if we don't care about any external implementers of the interfaces. Curious if anyone feels strongly that we shouldn't provide a default implementation.

Reply:
I think it's OK for now. If we decide to remove it (e.g. ahead of 1.0), it'd be no more breaking than not adding it in the first place would have been?